plugins/statuslines/lualine.nix

```diff
@@ -197,7 +264,7 @@ in
 
       winbar = mkEmptySectionOption "Winbar configuration";
```

Comment:
```suggestion
      winbar = mkEmptySectionOption "Winbar configuration used when active.";
```

Reply:
I dont think that's necessarily true. It would be used for active/inactive when inactive isn't defined.

Reply:
Ok. I guess either keep it simple and just add the missing `.`, or maybe attempt to explain how it interacts with `options.inactive_winbar`?

Reply:
This is what I had in my config to disable parts of lualine depending on where i'm at. 
```nix
    disabledFiletypes = {
      statusline = [
        # "neo-tree"
        "startify"
      ];
      winbar = [
        "aerial"
        "dap-repl"
        "neo-tree"
        "neotest-summary"
        "startify"
      ];
    };
```